src/HooglePlus/FilterTest.hs

```diff
@@ -20,59 +20,44 @@ import HooglePlus.Utils
 import Types.Environment
 import Types.Program
 import Types.Type hiding (typeOf)
+import Types.Filtering
 import Synquid.Type
 
 excludeFunctions :: [String]
-excludeFunctions = ["GHC.List.iterate"]
```

Comment:
I thought we don't need these with the `timeout` call you already have?

Reply:
`timeout` won't prevent the interpreter from printing infinite lists; i'll try to see if turning off outputs helps. Maybe we could only kill the thread once the next step is done.

Reply:
Resolved. Now we're based on the timeout function